src/definitions/constants.ts

```diff
@@ -30,4 +31,23 @@ export const COMMAND_AND_PROJECT_TYPE_MAP: { [command: string]: string[] } = {
     "surefire":[LIBERTY_MAVEN_PROJECT, LIBERTY_MAVEN_PROJECT_CONTAINER],
     "gradle":[ LIBERTY_GRADLE_PROJECT, LIBERTY_GRADLE_PROJECT_CONTAINER],
     "liberty.dev.debug": [LIBERTY_MAVEN_PROJECT, LIBERTY_GRADLE_PROJECT, LIBERTY_MAVEN_PROJECT_CONTAINER, LIBERTY_GRADLE_PROJECT_CONTAINER],
-};
\ No newline at end of file
+};
+
+export const COMMAND_TITLES = new Map();
+COMMAND_TITLES.set(localize("commands.title.refresh"), "liberty.explorer.refresh");
+
+COMMAND_TITLES.set(localize("commands.title.start"), "liberty.dev.start");
+COMMAND_TITLES.set(localize("commands.title.start.custom"), "liberty.dev.custom");
+COMMAND_TITLES.set(localize("commands.title.start.in.container"), "liberty.dev.start.container");
+
+COMMAND_TITLES.set(localize("commands.title.debug"), "liberty.dev.debug");
+
+COMMAND_TITLES.set(localize("commands.title.stop"), "liberty.dev.stop");
+
+COMMAND_TITLES.set(localize("commands.title.add.project"), "liberty.dev.add.project");
+COMMAND_TITLES.set(localize("commands.title.remove.project"), "liberty.dev.remove.project");
```

Comment:
Minor suggestion - can we move the add/remove commands to the bottom of the list?